charts/hmc-cft-hearing-service/Chart.yaml

```diff
@@ -3,14 +3,13 @@ appVersion: "1.0"
 description: A Helm chart for hmc-cft-hearing-service App
 name: hmc-cft-hearing-service
 home: https://github.com/hmcts/hmc-cft-hearing-service
-version: 1.6.8
+version: 1.6.9
 maintainers:
   - name: HMCTS CCD Dev Team
 dependencies:
   - name: java
     version: 3.7.3
     repository: 'https://hmctspublic.azurecr.io/helm/v1/repo/'
-  - name: servicebus
-    version: 0.4.0
+  - name: hmc
+    version: 0.0.2
```

Comment:
Note this will need a `condition` otherwise will attempt to run up as part of other env deployments too